.github/workflows/add-sponsors.yml

```
@@ -0,0 +1,89 @@
+      
+      - name: Update README
+        if: steps.fetch-sponsors.outputs.sponsor_count > 0
+        run: |
+          if [ -s sponsors.md ]; then
+            echo "Found sponsors data:"
+            cat sponsors.md
+            
+            # Create a temporary README with updated sponsors
+            awk -v new_content="$(cat sponsors.md)" '
+              BEGIN { in_sponsor_section = 0; printed = 0 }
+              /<!-- sponsors-start -->/ { 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)